dffml/df/base.py

```diff
@@ -234,7 +287,7 @@ def op(*args, imp_enter=None, ctx_enter=None, config_cls=None, **kwargs):
     ...     },
     ...     definition=canVote.op.inputs["p"],
     ... )
-    Input(value={'Bob': Person(name='Bob', age=19), 'Alice': Person(name='Alice', age=21), 'Mark': Person(name='Mark', age=90)}, definition=canVote.p)
+    Input(value={'Bob': Person(name='Bob', age=19), 'Alice': Person(name='Alice', age=21), 'Mark': Person(name='Mark', age=90)}, definition=canVote.inputs.p)
```

Comment:
We need another example after this one showing how it can auto-create a definition for the return type of an operation

Reply:
sure

Reply:
```python
    >>> @op
    ... def canVote(p: Dict[str, Person]) -> Dict[str, Person]:
    ...     return {
    ...         person.name: person
    ...         for person in filter(lambda person: person.age >= 18, p.values())
    ...     }
    ...
    >>>
    >>> Input(
    ...     value={
    ...         "Bob": {"name": "Bob", "age": 19},
    ...         "Alice": {"name": "Alice", "age": 21},
    ...         "Mark": {"name": "Mark", "age": 90},
    ...     },
    ...     definition=canVote.op.inputs["p"],
    ... )
    Input(value={'Bob': Person(name='Bob', age=19), 'Alice': Person(name='Alice', age=21), 'Mark': Person(name='Mark', age=90)}, definition=canVote.inputs.p)
    >>>
    >>> Input(
    ...     value={
    ...         "Bob": {"name": "Bob", "age": 19},
    ...         "Alice": {"name": "Alice", "age": 21},
    ...         "Mark": {"name": "Mark", "age": 90},
    ...     },
    ...     definition=canVote.op.outputs["result"],
    ... )
    Input(value={'Bob': Person(name='Bob', age=19), 'Alice': Person(name='Alice', age=21), 'Mark': Person(name='Mark', age=90)}, definition=canVote.outputs.result)
```

Reply:
thanks